### ISSUE TITLE
[SnippetBundle] Copy locale not copy all the content of a Snippet

### ISSUE THREAD

**Issue by plozmun** [sulu/sulu#5829]
| Q | A
| --- | ---
| Sulu Version | 2.2.4
| PHP Version | 7.4
| DB Version | MySQL 5.7
| Browser Version | Chrome

# Actual Behavior

When you copy a Snippet from a previous locale created, only the title is copied

# Expected Behavior

All the content including image reference must be copied like the pages

# Steps to Reproduce

- Go to sulu.rocks/admin
- Create a new Snippet  ( en) and fill title and description and save it
- Create a new translation ( de ) and select previous locale ( en ) 

# Possible Solutions

I don't know why but in ContentMapper when this is executed :
```
 $destDocument->getStructure()->bind($document->getStructure()->toArray());
```
$document->getStructure()->toArray() is an array with keys only

**Comment by niklasnatter** [comment]
Thanks for reporting! I suspect that this has something to do with the changes in #5573. The `SnippetController` probably needs to be adjusted similar to the `PageController`.

**Comment by plozmun** [comment]
@nnatter PR opened . It works :)

**Comment by niklasnatter** [comment]
Fixed by #5832 - thanks a lot for reporting and also fixing this! Its a pleasure to get community contributions like this 🙂
